test/integration/goldens/logging/LoggingServiceV2Client.java

```diff
@@ -435,7 +435,13 @@ public final WriteLogEntriesResponse writeLogEntries(WriteLogEntriesRequest requ
    *
    * <pre>{@code
    * try (LoggingServiceV2Client loggingServiceV2Client = LoggingServiceV2Client.create()) {
-   *   // Note: Not implemented yet, placeholder for paged unary rpc method sample code.
+   *   List<String> resource_names = new ArrayList<>();
+   *   String filter = "filter-1274492040";
```

Comment:
The samples currently use empty strings, existing example [here](https://github.com/googleapis/java-logging/blob/master/google-cloud-logging/src/main/java/com/google/cloud/logging/v2/LoggingClient.java#L292). Consider detecting this case at the default value call site, or if this kind of different behavior occurs often,  having a separate sample-specific variant of the default value method(s).